src/window/mod.rs

```diff
@@ -18,6 +18,7 @@ use tokio::sync::mpsc::UnboundedReceiver;
 
 #[cfg(target_os = "macos")]
 use glutin::platform::macos::WindowBuilderExtMacOS;
+use crate::settings::draw_background;
```

Comment:
If you intended to have this import only active if the target OS is `macos`, note that `#[...]` attributes only apply to the next item.

Reply:
I used code below to import the function only on macOS.
Thank you!

```rust
#[cfg(target_os = "macos")]
mod draw_background;
```